tamr_toolbox/dataset/_profile.py

```diff
@@ -0,0 +1,57 @@
+"""
```

Comment:
I would suggest this file be called `_dataset.py` since it will contain any functions we want namespaced under `tamr_toolbox.dataset`.  The function name is still good.

Reply:
Will make the change in the next commit.